brain/circuit.cpp

```diff
@@ -98,19 +101,15 @@ neuron::Morphologies Circuit::loadMorphologies( const GIDSet& gids,
     GIDSet::const_iterator gid = gids.begin();
     for( size_t i = 0; i < uris.size(); ++i, ++gid )
```

Comment:
I know but simply move ++gid inside the loop, and that gets rid of the GIDSet::const_iterator. Anyway, just a suggestion there.